package.json

```diff
@@ -87,6 +88,10 @@
 			{
 				"command": "reactNative.reloadApp",
 				"title": "React Native: Reload App"
+			},
+			{
+				"command": "reactNative.runInspector",
+				"title": "React Native: Element Inspector"
```

Comment:
I would change this to "React Native: Run Element Inspector" for more transparency to user.

Reply:
Yes, I think it can improve the user experience.